runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -188,6 +188,9 @@
 
 	private static final Pattern PICKPOCKET_REGEX = Pattern.compile("You pick (the )?(?<target>.+)'s? pocket.*");
 
+	private static final Pattern BIRDNEST_REGEX = Pattern.compile("You take (a|an) (.+) out of the bird's nest\\.");
```

Comment:
Mostly took inspiration from the pickpocket regex above w/ the exception of not having a named capture group since we don't care about the seed from the message itself due to the snapshot function handling that for us.